oresat_dxwifi/services/oresat_live.py

```diff
@@ -75,11 +78,32 @@ def capture(self):
 
         try:
             self.camera.create_videos()
+            self.state = State.STANDBY
         except Exception as error:
             self.state = State.ERROR
             logger.error("Something went wrong with camera capture...")
             logger.error(error)
 
+    def transmit(self):
+        self.state = State.TRANSMISSION
+
+        files = listdir(self.VIDEO_OUTPUT_DIRECTORY)
+
+        # Ideally, we could just pass in the output directory path to the Transmitter
+        # but in practice multi-file transmission to multi-file receiving has been inconsistent
+
+        for x in files:
+            x = self.VIDEO_OUTPUT_DIRECTORY + "/" + x
+            try:
+                tx = Transmitter(x)
+                p = Process(target=tx.transmit)
```

Comment:
Include a comment about why you're using multiprocessing here; it's not obvious.